Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Sharing/ISpatialLocalizer.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.MixedReality.Experimental.SpatialAlignment.Common;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
+{
+    public interface ISpatialLocalizer
+    {
+        Guid SpatialLocalizerId { get; }
+
+        bool TryDeserializeSettings(BinaryReader reader, out ISpatialLocalizationSettings settings);
+
+        ISpatialLocalizationSession CreateLocalizationSession(ISpatialLocalizationSettings settings);
```

Comment:
>CreateLocalizationSession [](start = 36, length = 25)

This should probably be a TryCreateLocalizationSession. If a session already exists it may not create one. If the settings aren't the expected type it may fail to create one as well.